### ISSUE TITLE
MultipleStatementAlignmentSniff gives misleading error message for multiple assignments on one line

### ISSUE THREAD

**Issue by pprkut** [squizlabs/PHP_CodeSniffer#480]
Generic.Formatting.MultipleStatementAlignment does not really cope well when multiple assignments are on the same line.

```
$a = $b = 1;
```
produces
```
Equals sign not aligned with surrounding assignments; expected 6 spaces but found 1 space
```

When the assignments are separated with a semicolon it looks a bit better, but still confusing.

```
$c = 1; $d = 1;
```
produces
```
Equals sign not aligned with surrounding assignments; expected 9 spaces but found 1 space
Each PHP statement must be on a line by itself
```

**Comment by aik099** [comment]
I guess we should ignore cases with several operators in same line. Or define some specific behavior, e.g. only one operation on a line or don't allow several operations together (like in your first example).

**Comment by pprkut** [comment]
Agreed. Ignoring seems like the best option. One can leave reporting the error that they shouldn't be on the same line/combined to a different sniff. Potentially DisallowMultipleStatementsSniff, which is where the additional error message for my second example comes from.

**Comment by gsherwood** [comment]
I just fixed this, but on https://github.com/squizlabs/PHP_CodeSniffer/issues/564
